horace_core/sqw/PixelData/@PixelData/private/binary_op_double_.m

```diff
@@ -57,8 +57,7 @@
     % object.
     %
     [npix_chunks, idxs] = split_vector_fixed_sum(npix(:), obj.base_page_size, npix_cum_sum);
-    page_number = 1;
-    while true
+    for page_number = 1:numel(npix_chunks)
```

Comment:
This is fundamentally changing the operation, was page_number not updating before?

Reply:
Yep, I'd forgotten to increment `page_number` when I refactored the function